egs/tedlium/run.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -e
+export POCOLM_ROOT=$(cd ../..; pwd -P)
+export PATH=$PATH:$POCOLM_ROOT/scripts
+
+num_dev_sentences=15000
+num_tedlium1_sentences=250000
+vocab_size=100000
+
+time_start=`date +%s`
+if [ ! -d cantab-TEDLIUM ]; then
+    echo "Downloading \"http://cantabresearch.com/cantab-TEDLIUM.tar.bz2\". "
+    wget --no-verbose --output-document=- http://cantabresearch.com/cantab-TEDLIUM.tar.bz2 | bzcat | tar --extract --file=- || exit 1
+    gzip cantab-TEDLIUM/cantab-TEDLIUM-pruned.lm3
+    gzip cantab-TEDLIUM/cantab-TEDLIUM-unpruned.lm4
+fi
+
+if [ ! -d data ]; then
+  mkdir data
+  mkdir data/text
+fi
+
+cat cantab-TEDLIUM/cantab-TEDLIUM.txt | tail -n $num_dev_sentences | sed 's/ <\/s>//g'  > data/text/dev.txt
```

Comment:
These two lines should be:

```
 tail -n +$num_dev_sentences <cantab-TEDLIUM/cantab-TEDLIUM.txt | sed 's/ <\/s>//g'  > data/text/cantab_tedlium.txt
head -n $num_dev_sentences <cantab-TEDLIUM/cantab-TEDLIUM.txt | sed 's/ <\/s>//g'  > data/text/dev.txt
```
best not to use cat, it's a wasted process, use redirection (<).
tail -n +X gives you all but the first X lines of a file.
Don't call it tedlium1 because I think that's not what it is, it's a dataset that cantab got from somewhere.